core/src/main/java/org/bitcoinj/evolution/SimplifiedMasternodeList.java

```diff
@@ -80,7 +75,7 @@ public SimplifiedMasternodeList(NetworkParameters params) {
 
     @Override
     protected void parse() throws ProtocolException {
-        if (protocolVersion >= params.getProtocolVersionNum(NetworkParameters.ProtocolVersion.BLS_SCHEME)) {
+        if (protocolVersion >= NetworkParameters.ProtocolVersion.BLS_SCHEME.getBitcoinProtocolVersion()) {
```

Comment:
For serialization, we need to use particular protocol versions that don't change and cannot be overridden as the serialization must match Dash Core 19.2 exactly.